src/components/Login.tsx

```diff
@@ -43,7 +45,7 @@ const Login = () => {
 
     const login = useCallback(
         (token: JWTToken) => {
-            dispatch(actions.setToken(token))
+            dispatch(actions.setToken(token, false))
```

Comment:
cannot we import just the action we eed to use rather than importing the whole file?